src/src/rx_main.cpp

```diff
@@ -1044,7 +1044,7 @@ static void ICACHE_RAM_ATTR updateSwitchModePendingFromOta(uint8_t newSwitchMode
 static bool ICACHE_RAM_ATTR ProcessRfPacket_SYNC(uint32_t const now, OTA_Sync_s const * const otaSync)
 {
     // Verify the first two of three bytes of the binding ID, which should always match
```

Comment:
I think the comment here needs to change. No longer 2 bytes.

Reply:
Yep, and there are a few other places that need cleaning up re magic numbers etc.  Once https://github.com/ExpressLRS/ExpressLRS/pull/2932 is merged and the overall functionality has been decided Ill clean it all up.